scripts/gen-genesis.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+REGEN_HOME="/tmp/regen$(date +%s)"
+CHAIN_ID=regen-1
+
+set -e
+
+echo "...........Init Regen.............."
+
+git clone https://github.com/regen-network/regen-ledger
+cd regen-ledger
+git checkout v1.0.0-rc3
+make build
+chmod +x ./build/regen
+
+./build/regen init --chain-id $CHAIN_ID validator --home $REGEN_HOME
+
+echo "..........Fetching genesis......."
+rm -rf $REGEN_HOME/config/genesis.json
+curl -s https://raw.githubusercontent.com/regen-network/mainnet/main/$CHAIN_ID/genesis-prelaunch.json >$REGEN_HOME/config/genesis.json
```

Comment:
I think this would be cleaner if it pulls from `../regen-1/genesis-prelaunch.json` in the current directory / repo, as opposed to pulling from github again.